extensions/sql-database-projects/src/models/tree/databaseReferencesTreeItem.ts

```diff
@@ -6,14 +6,17 @@
 import * as vscode from 'vscode';
 import * as path from 'path';
 import * as constants from '../../common/constants';
-import { BaseProjectTreeItem, MessageTreeItem } from './baseTreeItem';
+
```

Comment:
Nit: Remove extra line.

Reply:
That was actually intentional to break up the different styles of imports.  Makes it easier to scan and find specific imports, I think.